packages/ts/generator-core/src/PluginManager.ts

```diff
@@ -2,13 +2,14 @@ import type LoggerFactory from '@vaadin/hilla-generator-utils/LoggerFactory.js';
 import type Plugin from './Plugin.js';
 import type { PluginConstructor } from './Plugin.js';
 import type ReferenceResolver from './ReferenceResolver.js';
-import type SharedStorage from './SharedStorage.js';
+import type { SharedStorage } from './SharedStorage.t.js';
 
 export default class PluginManager {
   readonly #plugins: Plugin[];
 
   constructor(plugins: readonly PluginConstructor[], resolver: ReferenceResolver, logger: LoggerFactory) {
     const standardPlugins = [
+      'TransferTypesPlugin',
```

Comment:
I suggest adding a comment here to make sure this stays as the first plugin to apply, unless something else should intentionally come before.